server/.gitignore

```diff
@@ -0,0 +1 @@
+coverage.txt
```

Comment:
Probably we should include here the `dist` directory.

Reply:
@jespino, this is odd -- the starter template [has this folder excluded](https://github.com/mattermost/mattermost-plugin-starter-template/blob/master/.gitignore) in the root `.gitignore`, but I don't see that file at all in this repository. @iomodo, any ideas?

Reply:
My bad, copied wrongly in the forked repo. I'll commit the fix.